src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefault.java

```diff
@@ -4,19 +4,44 @@
     public void foo() {
         int i = 1;
         switch (i) {
-        case 1: i++; break;
-        case 2: i--; break;
-        default: return;
+            case 1: i++; break;
```

Comment:
Why did test cases in inputs change?

Reply:
I added tab spaces to give a consistent and even look to whole file, the already added inputTestCases were not changed by me.

Reply:
please move any unrelated to issue fixes (formatting, indentations to separate commit of the same PR)